keps/sig-storage/1698-generic-ephemeral-volumes/kep.yaml

```diff
@@ -14,7 +14,7 @@ approvers:
 prr-approvers:
   - "@wojtek-t"
 stage: beta
```

Comment:
You want to change to stable here (and add corresponding entry in production-readiness).

Reply:
Done.